test/common/runtime/runtime_impl_test.cc

```diff
@@ -35,12 +36,30 @@ TEST(UUID, sanityCheckOfUniqueness) {
 
 class RuntimeImplTest : public testing::Test {
 public:
+  void SetUp() override {
+    // Some ugly ::system escaping. This is needed as this test relies on
+    // symlinks which can only exist in the test temporary directory to play
+    // nice with Bazel, so we create them on-the-fly. Could do this cleanly with
+    // std::filesystem if/when we have C++17.
+    ASSERT_EQ(0, ::system(("cp -rf --parents test/common/runtime/test_data " +
```

Comment:
Yeah, it's a precondition that `$TEST_TMPDIR` is empty, Bazel will provide this. I've updated the test script to pick a fresh tmp dir each time by default to ensure this is true in the no touch case. If someone feeds in explicit `$TEST_TMPDIR`, the documentation says they should clean it.

I rather do it this way than do an explicit `rm -rf`, which can get very dangerous when fat fingering the `$TEST_TMPDIR`value.